py2pack/templates/opensuse.spec

```diff
@@ -25,10 +25,12 @@ Group:          Development/Languages/Python
 Source:         {{ source_url|replace(version, '%{version}') }}
 BuildRequires:  python-devel {%- if requires_python %} = {{ requires_python }} {% endif %}
 BuildRequires:  python-setuptools
+{%- if install_requires and install_requires != none %}
```

Comment:
Sorry for the silly questions, but is None? Also if `install_requires` is evaluated as `False`, can `None` be evaluated as `True`? (I know one case in `lxml`, but not sure here)